api/src/pcapi/core/bookings/repository.py

```diff
@@ -883,27 +883,27 @@ def find_individual_bookings_event_happening_tomorrow_query() -> list[Individual
     tomorrow = datetime.utcnow() + timedelta(days=1)
     tomorrow_min = datetime.combine(tomorrow, time.min)
     tomorrow_max = datetime.combine(tomorrow, time.max)
+
     return (
-        IndividualBooking.query.join(Booking, Booking.stock, Stock.offer, Offer.venue)
+        IndividualBooking.query.join(
+            IndividualBooking.user,
+            IndividualBooking.booking,
+            Booking.stock,
+            Stock.offer,
+            Offer.venue,
+        )
+        .join(Booking.activationCode, isouter=True)
```

Comment:
(micro-détail / question de goût) je trouve plus lisible la méthode `outerjoin()` : 

```suggestion
        .outerjoin(Booking.activationCode)
```

... mais c'est probablement une question de goût. (C'est juste au cas où vous ne connaissiez pas.)